scripts/ai-help-macros.ts

```diff
@@ -56,7 +57,14 @@ export async function updateEmbeddings(
   }
 
   // Supabase.
```

Comment:
Super tiny little nitty-nit: At this point, it is not about Supabase any more, it should be Postgres.